Engine/source/math/mPoint2.h

```diff
@@ -285,6 +288,13 @@ inline Point2I::Point2I(S32 _x, S32 _y)
 }
 
 
+inline Point2I::Point2I(S32 xy)
+   : x(xy), y(xy)
+{
+   //
```

Comment:
Why do these empty comments exist? Why not have it an entirely empty function like what you did in mPoint4.h? I find it cleaner that way.